assets/js/stripe-payment-request.js

```diff
@@ -1,5 +1,5 @@
 /* global wc_stripe_payment_request_params, Stripe */
-jQuery( function( $ ) {
+jQuery( function($ ) {
```

Comment:
Just double checking if this is the right formatting? I would expect the previous one TBH :)

Reply:
indeed. thx